unit_tests/mocks/services/family_service.py

```diff
@@ -23,10 +29,23 @@ def mock_get_family(import_id: str) -> Optional[FamilyReadDTO]:
             return create_family_dto(import_id)
 
     def mock_search_families(q_params: dict) -> list[FamilyReadDTO]:
+        VALID_PARAMS = [
```

Comment:
Hmmm, would it be better to import `VALID_PARAMS` (if we make them accessible? Otherwise the test for valid params doesn't feel like its testing the actual code :thinking: